index.js

```diff
@@ -119,16 +100,81 @@
   //  $$type :: String
   var $$type = '@@type';
 
-  //  type :: Any -> String
+  //  RPARSE :: RegExp
+  var RPARSE = /^([^]+)\/([^]+?)(?:@(\d+))?$/;
```

Comment:
`.` excludes newline characters, `[^]` is the best performing way to express "all characters including newlines"